MediaBrowser.Model/ApiClient/ServerDiscoveryInfo.cs

```diff
@@ -1,32 +1,57 @@
-#nullable disable
-#pragma warning disable CS1591
-
 namespace MediaBrowser.Model.ApiClient
 {
+    /// <summary>
+    /// The server discovery info model.
+    /// </summary>
     public class ServerDiscoveryInfo
     {
         /// <summary>
-        /// Gets or sets the address.
+        /// Initializes a new instance of the <see cref="ServerDiscoveryInfo"/> class.
+        /// </summary>
+        /// <param name="address">The server address.</param>
+        /// <param name="id">The server id.</param>
+        /// <param name="name">The server name.</param>
+        public ServerDiscoveryInfo(string address, string id, string name)
+        {
+            Address = address;
+            Id = id;
+            Name = name;
+            EndpointAddress = null;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ServerDiscoveryInfo"/> class.
+        /// </summary>
+        /// <param name="address">The server address.</param>
+        /// <param name="id">The server id.</param>
+        /// <param name="name">The server name.</param>
+        /// <param name="endpointAddress">The endpoint address.</param>
+        public ServerDiscoveryInfo(string address, string id, string name, string endpointAddress)
```

Comment:
_Endpoint_ is not used anywhere. If it doesn't break API, it could be removed.

https://github.com/jellyfin/jellyfin/blob/c2af32d035352d333c3e2852d25fde51f984006b/Emby.Server.Implementations/Udp/UdpServer.cs#L56